lxd/lifecycle/lifecycle_network.go

```diff
@@ -0,0 +1,35 @@
+package lifecycle
```

Comment:
File can probably be `network.go`, `lifecycle_` is a bit redundant given it's the package name :)